src/snowflake/cli/plugins/nativeapp/codegen/snowpark/python_processor.py

```diff
@@ -224,7 +224,7 @@ def process(
 
                 with open(sql_file, "a") as file:
                     file.write(
-                        f"-- Generated by the Snowflake CLI from {relative_py_file}\n"
+                        f"\n-- Generated by the Snowflake CLI from {relative_py_file}\n"
```

Comment:
but why? This just adds a blank line at the top of every SQL file we generate. It just looks odd.